fmriprep/workflows/fieldmap/fmap.py

```diff
@@ -127,12 +124,14 @@ def init_fmap_wf(reportlets_dir, omp_nthreads, fmap_bspline, name='fmap_wf'):
 
 
 def _torads(in_file, out_file=None):
+    import os
     from math import pi
     import nibabel as nb
-    from fmriprep.utils.misc import genfname
+    from niworkflows.nipype.utils.filemanip import fname_presuffix
 
     if out_file is None:
-        out_file = genfname(in_file, suffix='rad')
+        out_file = fname_presuffix(
+            in_file, suffix='rad', newpath=os.getcwd())
```

Comment:
Yes, that's what I was suggesting. As I [said](https://github.com/poldracklab/fmriprep/pull/640#discussion_r131379154), it's no big deal. I'm happy to merge without that change.